configs/oasisengine.json

```diff
@@ -16,15 +17,13 @@
       "variables": {
         "version": [
           "0.3",
-          "0.4"
+          "0.4",
+          "0.5"
         ]
       },
       "page_rank": 1
     }
   ],
-  "sitemap_urls": [
-    "https://oasisengine.cn/sitemap.xml"
-  ],
```

Comment:
Having sitemaps is always better, even if they don't contain all the URLs it's still a good entry point for the crawler

Reply:
Thank you for your advice. I've recovered the code.

Reply:
Feel free to merge the master branch so I can merge

Reply:
@shortcuts Pls merge to master branch🥳

Reply:
@eyworldwide it seems I'm not able to merge master in your PR due to some collaborating settings on your GH account

Reply:
@shortcuts I'll check the problem first. If it not works, I will try to pull another request through my personal account.

Reply:
I've pulled another PR from a new branch: https://github.com/algolia/docsearch-configs/pull/4556